jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -694,6 +693,58 @@ public Response getInvoicesForAccount(@PathParam("accountId") final UUID account
         return Response.status(Status.OK).entity(result).build();
     }
 
+    @TimedResource
+    @GET
+    @Path("/{accountId:" + UUID_PATTERN + "}/" + INVOICES + "/" + PAGINATION)
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "Retrieve account invoices", response = InvoiceJson.class, responseContainer = "List")
+    @ApiResponses(value = {@ApiResponse(code = 400, message = "Invalid account id supplied"),
+                           @ApiResponse(code = 404, message = "Account not found")})
+    public Response getInvoicesForAccountWithPagination(@PathParam("accountId") final UUID accountId,
```

Comment:
Could we just nam it to `getInvoicesForAccount`? The name here will dictate the Java client API.

Reply:
Using the `getInvoicesForAccount` name causes an issue in generating the next page URI [here](https://github.com/reshmabidikar/killbill/blob/c58d2684202706005f97cf3ef07daf21c73930db/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java#L730) since there is another `getInvoicesForAccount` method [here](https://github.com/reshmabidikar/killbill/blob/c58d2684202706005f97cf3ef07daf21c73930db/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java#L651). Would you like me to look into the logic of `uriBuilder.nextPage` to see if there is any way to work around this?

Reply:
oh, I see. Never mind then!